### PR TITLE
make text able to got to 3

### PR DESCRIPTION
**Pull request by MrClogsworthYT**
title thing go brrrr
but seriously 
so you know how people sometimes try to add 3 texts at the same time in the text file for title text? this allows them to go 1 more! just enough time to add mod name or credits!

### CODE REVIEW

**Comment by MrClogsworthYT** [comment]
btw I couldn't test this

**Comment by MrClogsworthYT** [comment]
so I expect atleast 1 bug

File: source/TitleState.hx [hunk 1]
```diff
@@ -312,6 +313,7 @@ class TitleState extends MusicBeatState
 			coolText.y += (textGroup.length * 60) + 200 + offset;
 			credGroup.add(coolText);
 			textGroup.add(coolText);
+			textGroup.add(coolText):
```
Comment (l1ttleO) [inline source/TitleState.hx:316]:
```suggestion
			textGroup.add(coolText);
```

**Comment by l1ttleO** [comment]
This pull request does nothing. You are adding text to `textGroup`, but if you look at the code, that group is created, but **never added**, which means it doesn't exist
Adding this group after it's creation, along with changes made in this PR, only results in the `Friday Night Funkin'` text not disappearing when you get to the `Press Enter to begin` screen (because the `skipIntro()` function that is called after the 16th beat removes __only__ the `credGroup`).

The proper way to do this would be to use `addMoreText` with `curWacky[2]`

Moreover, I think this should be contributed to Psych Engine instead, as it makes changes to the *modding* process (creating mods), while ProjectFNF focuses to spice up your *playing* process (playing the game and mods). This policy will apply to all changes made to ProjectFNF.

**Pull request closed without merging**